modules/nw-ingress-controller-configuration-proxy-protocol.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-configuration-proxy-protocol_{context}"]
+= Configuring PROXY protocol for an Ingress Controller
```

Comment:
We can verify, but I think we can lower-case proxy

Reply:
"PROXY protocol" is used in the protocol specification; see http://www.haproxy.org/download/2.2/doc/proxy-protocol.txt for reference.  (It does look shouty though...)